.github/labeler.yml

```diff
@@ -38,12 +38,29 @@
 "NPC / Factions":
   - "**/npcs/**/*"
 
+"Info / User Interface":
+  - "**/ui/**/*"
+  - "**/widget/**/*"
+  - "src/list_view.*"
+  - "src/surroundings_menu.*"
+
 "Appliance/Power Grid":
   - "**/appliance**"
 
 "Monsters":
   - "**/monsters/**/*"
 
+  "Lore":
+    - data/json/npcs/BG_trait_groups.json
+    - data/json/npcs/Personality_trait_groups.json
+    - data/json/npcs/personality_traits.json
+    - "**/backgrounds/**/*"
+    - "**/snippets/**/*"
```

Comment:
```suggestion
"Lore":
  - data/json/npcs/BG_trait_groups.json
  - data/json/npcs/Personality_trait_groups.json
  - data/json/npcs/personality_traits.json
  - "**/Backgrounds/**/*"
  - "**/snippets/**/*"
```

Reply:
ah so it's case dependent?  I had wondered that.